### DISCUSSION TITLE
PyInstaller trouble with PIL.Image and or infi.systray

### DISCUSSION THREAD

**Discussion by NiceeTV** [pyinstaller/pyinstaller#7365] [Help]
I've been trying to make an app that acts as a system tray icon that when clicked, opens tkinter window and closes when clicked again, however the module infi.systray that I use is not suited for what I want, therefore I added a few tweaks. Even though I believe it has nothing with my problem, I think it is better to add it.

PyInstaller itself has no trouble with tkinter apps that I tried, but when I add PIL.Image and load some picture, icon in my case, it throws error below. At first I thought that it has a problem with PIL and have tried many things such as hidden_imports and such. Then I tried with only infi.systray. However the result is the same.

I'll add modified infi.systray file and my minimalistic code that recreates the problem.
File that I want to convert is named "7.3.py", I use icon file as system tray icon. File "trayback" is from module infi.systray that has been modified. Even tough PIL is in this program useless, it may be connected to the problem as it is a vast module with many files.

Changes: I removed the compulsory Quit button, when an option has empty title, it is not drawn, when menu has no options (empty title options are excluded), it is not drawn. I changed Left Double Click command to Left Single Click. But as I said, it might not have an effect on it, but for recreation purposes I should add that.

I've been racking my brain for the second day, cruising through forums with no avail, so I came here in hopes of solving the issue finally.

error message:

![image](https://user-images.githubusercontent.com/88597437/211011503-c56ceb8f-5ad1-4ce0-8643-0dcffc973ce8.png)
![image](https://user-images.githubusercontent.com/88597437/211011566-6984787e-7c01-4254-9080-f73f804a1dd4.png)
![image](https://user-images.githubusercontent.com/88597437/211011618-71a566b9-0dc6-4185-9765-56b56e380af7.png)

[New folder.zip](https://github.com/pyinstaller/pyinstaller/files/10360029/New.folder.zip)

**Comment by bwoodsend** [comment]
It looks like you're using an old version of Python provided by the Windows Store. The Windows store Python used to be a dysfunctional thing with its permissions so heavily locked down that PyInstaller wasn't even allowed read access to the Python environment.

Either way though, can we have the build logs as plain copy/paste text please. I can't even see which files are raising the exception because the filenames are truncated in your screenshot.
